### ISSUE TITLE
Enhancement suggestion: delete image action.

### ISSUE THREAD

**Issue by mcsimenc** [tzutalin/labelImg#454]

**Comment by 7thstorm** [comment]
Agree, as well as delete the xml/txt file when the label itself has been deleted.
currently, when you delete all labels, it keeps the xml file but removes the <name> tag

**Comment by ssbnsk99** [comment]
Another suggestion to add, it would be appreciable if there is a button to move the current image to another directory so that it could be useful to look at them and make analysis later.

**Comment by srikanth9444** [comment]
Shortcut for deleting current image (ctrl+shift+d) is not working

**Comment by nine** [comment]
@srikanth9444 What do you mean with 
> Shortcut for deleting current image (ctrl+shift+d) is not working

- Observed behavior
- Expected behavior
